molecule/python3-ubuntu1804/molecule.yml

```diff
@@ -0,0 +1,52 @@
+---
+dependency:
+  name: galaxy
+  options:
+    role-file: molecule/resources/requirements.yml
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: omero-server-python3
+    image: leandelivery/docker-systemd:ubuntu-18.04
+    command: /sbin/init
+    privileged: true
+    tmpfs:
+      - /sys/fs/cgroup
+    groups:
+      - docker-hosts
+      - omero-py3
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+  # To test the upgrade process without breaking the idempotence check
+  # - install omero during the prepare step which is only run once
+  # - attempt to upgrade in the converge step
+  options:
+    v: true
+    diff: true
+    # tags: [x]
+  playbooks:
+    # TODO: Use shared test playbooks
+    converge: ../resources/playbook-py3.yml
+  inventory:
+    host_vars:
+      omero-server-python3:
+        postgresql_version: "10"
```

Comment:
pg version 11 is the version we recommend so we should be testing it

Reply:
Updated